content/README.md

```diff
@@ -388,7 +388,7 @@ The homepage is the main Table of Contents file for the docs site. The homepage
 
 To create a product guides page (e.g. [Actions' Guide page](https://docs.github.com/en/actions/guides)), create or modify an existing markdown file with these specific frontmatter values:
 
-1. Use the product guides page template by referencing it `layout: product-guides`
+1. Use the product guides page template by referencing it's `layout: product-guides`
```

Comment:
It should be "its" but we generally try to avoid using pronouns as these can be ambiguous, especially for translated content. I am going to suggest the following:
```suggestion
1. Use the product guides page template by referencing `layout: product-guides` in the frontmatter.
```